doc/classes/Texture.xml

```diff
@@ -1,10 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Texture" inherits="Resource" version="4.0">
 	<brief_description>
+		The global C# namespace (Mono-enabled builds only).
```

Comment:
Welp... That was a bogus rebase :)

Reply:
Fixed in dfa5f88c11.

Reply:
Sorry, I didn't notice this in the rebase log. I was wondering where `@C#.xml` went.